### PR TITLE
[CPL][INPUT] Update he-IL.rc

### PR DESCRIPTION
**Pull request by omeritzics**

### CODE REVIEW

**Comment by HBelusca** [comment]
Cc @reactos/lang-hebrew

**Comment by omeritzics** [comment]
> Cc @reactos/lang-hebrew

Hello, I think @reactos/lang-hebrew isn't active because the page was not found.

**Comment by HBelusca** [comment]
@omeritzics : there are two people in the list who recently contributed: @peterooch and @shytzedaka .

**Comment by omeritzics** [comment]
Is there another way to reach the group list?

**Comment by binarymaster** [comment]
> the page was not found

@omeritzics this page is available only for team members, that's why you get 404.

File: dll/cpl/input/lang/he-IL.rc [hunk 1]
```diff
@@ -97,18 +97,18 @@ BEGIN
...
     IDS_LAYOUT "תצורת מקשים"
     IDS_REM_QUESTION "Do you want to remove selected keyboard layout?"
     IDS_CONFIRMATION "אישור"
-    IDS_LAYOUT_EXISTS "This layout already exists."
-    IDS_LAYOUT_EXISTS2 "This layout already exists and can not be added."
+    IDS_LAYOUT_EXISTS "הפריסה הזו כבר קיימת."
+    IDS_LAYOUT_EXISTS2 "הפריסה הזו כבר קיימת ואינה ניתנת להוספה."
     IDS_CTRL_SHIFT "Ctrl+Shift"
-    IDS_LEFT_ALT_SHIFT "Left Alt+Shift"
+    IDS_LEFT_ALT_SHIFT "מקש Alt השמאלי+Shift"
```
Comment (peterooch) [inline dll/cpl/input/lang/he-IL.rc:109]: I think "שמאלי" would be better than "השמאלי"

File: dll/cpl/input/lang/he-IL.rc [hunk 2]
```diff
@@ -97,18 +97,18 @@ BEGIN
...
     IDS_SWITCH_BET_INLANG "החלף בין שפות כתיבה"
-    IDS_REBOOT_NOW, "Reboot now?"
+    IDS_REBOOT_NOW, "לאתחל כעת?"
```
Comment (peterooch) [inline dll/cpl/input/lang/he-IL.rc:111]: I think "?להפעיל מחדש כעת" or something like that would be more clearer.

**Review by peterooch — CHANGES_REQUESTED** [review]
Some nitpicks, everything else is alright.

**Comment by omeritzics** [comment]
OK, changed it.

**Comment by HBelusca** [comment]
thanks!

**Pull request merged**